jakartaee9/JakartaEE9.md

```diff
@@ -23,9 +28,34 @@ These component release plans will utilize the existing Eclipse mechanism for de
 - [Jakarta Activation Framework 2.0 Release Plan](https://projects.eclipse.org/projects/ee4j.jaf/releases/2.0/plan)
 - [Jakarta Enterprise Beans 4.0 Release Plan](https://projects.eclipse.org/projects/ee4j.ejb/releases/4.0/plan)
 
-## Jakarta EE 9 Project board
-
-Jakarta EE 9 will track its progress with a [Github Project Board](https://github.com/orgs/eclipse-ee4j/projects/17).
+## Jakarta EE 9 schedule
+
+Overall schedule (dates are tentative and subject to change).  The waves are based off of the content of the [Jakarta EE 9 Release Plan](JakartaEE9ReleasePlan).
+
+- Milestone 1 release (Java 8) - **June 23** announce
+    - Development Complete - June 12
+    - Testing Complete - June 19
+- Final Jakarta EE 9 release (Java 11) - **Sept 29** announce
+    - Wave 0 Specification PRs Complete - July 21
+    - Wave 0 Specification Ballots Complete - Aug 4
```

Comment:
Arent wave 0 specs independent of the others and can be completed at any time? Could be sprinkled into any of the other wave batches? Just a suggestion

Reply:
True, But that's hard to put into a schedule...  If you have ideas on how to represent the Wave 0 drops, I'm open to suggestions.  By leaving it first in the proposed schedule, then it allows them to come in later...

Reply:
I did end up changing this slightly.  When I adjusted the final date to be consistent with the Livestream event, then Wave 0 just became too early in July.  So, I just indicated that Wave 0 has to be done sometime between now and the end of Wave 6.